frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -157,7 +165,7 @@ private void refreshModel(GitRepository gitRepository, List<String> skippedBranc
       if (gitMacheteRepositorySnapshot.getRootBranches().isEmpty()) {
         setTextForEmptyTable(
             /* upperText */ "Provided machete file (${macheteFilePath}) is empty.",
-            /* lowerText */ "Discover repository layout",
+            /* lowerText */ "Discover the repository's branch layout",
```

Comment:
Pls use properties for text

Reply:
Ok but I will must add a next dependency to this module - this is the reason why I didn't that

Reply:
Done

Reply:
`frontendResourceBundles` are lightweight + don't pull other modules transitively, okay to add